### PR TITLE
update repositories.toml

### PR DESCRIPTION
**Pull request by harisdev-netizen**
Issue: add dark mode to the project

#### ℹ️ Repository information

**The repository has**:

- [x] At least three issues with the `good first issue` label.
- [x] At least 10 contributors.
- [x] Detailed setup instructions for the project.
- [x] CONTRIBUTING.md
- [x] Actively maintained.

### CODE REVIEW

**Review by harisdev-netizen — COMMENTED** [review]
This is looking perfect and I like request changes to repositories.toml :)

File: data/repositories.toml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 repositories = [
+  'github.com/harisdev-netizen/connect.git'
```
Comment (harisdev-netizen) [inline data/repositories.toml:2]: Kindly push this repository to repositories.toml

Thank you ;)

**Review by harisdev-netizen — COMMENTED** [review]
Thank you.

**Comment by withshubh** [comment]
Closing this PR as the repo doesn't meet the required criteria.

**Pull request closed without merging**